tests/postgres_tests/migrations/0001_setup_extensions.py

```diff
@@ -22,9 +23,10 @@ class Migration(migrations.Migration):
         BtreeGinExtension(),
         # Ensure CreateExtension quotes extension names by creating one with a
         # dash in its name.
+        CITextExtension(),
```

Comment:
Can you please leave this line at the end or make sure that the comment above still goes with the `CreateExtension('uuid-ossp')` line.